languages/SMW_LanguageDe.php

```diff
@@ -96,8 +96,8 @@ class SMWLanguageDe extends SMWLanguage {
 		'_PVAP' => 'Erlaubt Muster',
 		'_DTITLE' => 'Anzeigetitel',
 		'_PVUC' => 'Erlaubt Wert einmal',
-		'_PEID' => 'External identifier',
-		'_PEFU' => 'External formatter uri',
+		'_PEID' => 'Kennung zur externen URI',
```

Comment:
@mwjames I am assuming here that this special property will always be used for exteral identifiers in connection with constructing an URI, so no other external ...

Yes, I am deliberately translating the special property and datatype differently. So no worries.